### ISSUE TITLE
XML

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1509]
**XML**

please provide a how to use an xml to view arcgis data and images explanation.

-----

System: Chrome 68.0.3440 / Windows 10.0.0
User: [See in registry](https://www.gbif.org/api/feedback/user/2433fe58d67fc95be6040854d4311e0e:a14bb65f37331c70c2f0c124111efb29764b6ed19f2a322250b2091c953a450c50a73f62648c468afeea9f4bcdd0a5ded922bccbf76dd47d89d329cf3f86d3b4)
Referer: https://www.gbif.org/faq?question=opening-gbif-csv-in-excel
Window size: width 1440 - height 714
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-08-22T21:40:37.049Z',mode:absolute,to:'2018-08-22T21:46:37.049Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-08-22T21:40:37.049Z',mode:absolute,to:'2018-08-22T21:46:37.049Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by ManonGros** [comment]
That is not the first time we get this request. An other user had trouble with arcgis before.

**Comment by MattBlissett** [comment]
#1507 was submitted by the same user (from "See in registry")

**Comment by ManonGros** [comment]
No, I meant I got a similar request in an email conversation (helpdesk).

**Comment by dnoesgaard** [comment]
@MattBlissett who do I need to bribe to get administrative rights to access this information?

**Comment by MattBlissett** [comment]
Please ask someone in Data Products or Informatics to give you an overview of the Registry console, and finish by granting you `REGISTRY_ADMIN` permission.
